examples/13_array_length/main.neva

```diff
@@ -0,0 +1,12 @@
+const n any = [4,123]
```

Comment:
If there's no difference between example and file in `e2e/tests`, remove file in tests and simply use example for e2e test